volumes/miovision/sql/table/create-mv-mio_cent.sql

```diff
@@ -107,7 +107,7 @@ SELECT
     dl.intersection_uid AS data_int,
     dl.leg AS data_leg
 FROM data_legs AS dl
-FULL JOIN miovision_api.centerline_miovision AS cm ON cm.intersection_uid = dl.intersection_uid AND cm.leg = dl.leg -- full join means I can see what's in each pile
+FULL JOIN miovision_api.centreline_miovision AS cm ON cm.intersection_uid = dl.intersection_uid AND cm.leg = dl.leg -- full join means I can see what's in each pile
```

Comment:
probs not in the scope of this PR but we should be referencing the gis_core centreline and intersection table meow